src/SFA.DAS.AANHub.Application/Apprentices/Commands/CreateApprenticeMember/CreateApprenticeMemberCommandHandler.cs

```diff
@@ -40,8 +46,19 @@ public class CreateApprenticeMemberCommandHandler : IRequestHandler<CreateAppren
             Resource = MembershipUserType.Apprentice
         });
 
+        var tokens = await GetTokens(command, cancellationToken);
+        Notification notification = NotificationHelper.CreateNotification(command.MemberId, EmailTemplateName.ApprenticeOnboardingTemplate, tokens, true);
+        _notificationsWriteRepository.Create(notification);
+
         await _aanDataContext.SaveChangesAsync(cancellationToken);
 
         return new ValidatedResponse<CreateMemberCommandResponse>(new CreateMemberCommandResponse(member.Id));
     }
+
+    private async Task<string> GetTokens(CreateApprenticeMemberCommand command, CancellationToken cancellationToken)
+    {
+        var region = await _regionsReadRepository.GetRegionById((int)command.RegionId!, cancellationToken);
```

Comment:
The other option here is to use command.RegiondId.GetValueOrDefault() to avoid casting and null negation